pkg/commands/oscommands/cmd_obj.go

```diff
@@ -15,6 +16,9 @@ type ICmdObj interface {
 	// into a terminal e.g. 'sh -c git commit' as opposed to 'sh -c "git commit"'
 	ToString() string
 
+	// outputs args vector e.g. ["git", "commit", "-m", "my message"]
+	ToArgs() []string
```

Comment:
I'd call this just `Args`. `ToArgs` sounds like some sort of computation is involved, but it's just a getter.

Reply:
Sounds good